docs-gen/content/rule_set/data_entry/data_units.md

```diff
@@ -81,8 +82,8 @@ Vehicle.Powertrain.Transmission.ClutchWear:
 ## Supported Data Units in VSS Standard Catalog
 
 The VSS syntax does not in itself specify what units can be used, the unit attribute as declared for signals in *.vspec files is optional and can contain an arbitrary string value.
-[VSS-Tools](https://github.com/COVESA/vss-tools) however require that all units used are defined,
-and at is also a requirement for signals in the VSS standard catalog.
+[VSS-Tools](https://github.com/COVESA/vss-tools) however require that all units used are defined
+and therefore is also a requirement for signals in the VSS standard catalog.
```

Comment:
Reading this, I feel we should just demand a unit in VSS for it to be valid  and be done with it.

Not related to this PR though